src/frontends/pytorch/src/transforms/aten_stack_list_construct_replacer.cpp

```diff
@@ -0,0 +1,58 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "aten_stack_list_construct_replacer.hpp"
+
+#include "openvino/core/rt_info.hpp"
+#include "openvino/op/util/framework_node.hpp"
+#include "openvino/pass/pattern/matcher.hpp"
+#include "openvino/pass/pattern/op/wrap_type.hpp"
+#include "utils.hpp"
+
+namespace ov {
+namespace frontend {
+namespace pytorch {
+namespace pass {
+
+AtenStackListConstructReplacer::AtenStackListConstructReplacer() {
+    auto stack = ov::pass::pattern::wrap_type<ov::op::util::FrameworkNode>();
```

Comment:
```suggestion
    auto list_construct = ov::pass::pattern::wrap_type<ov::op::util::FrameworkNode>();
    auto axis = ov::pass::pattern::wrap_type<opset10::Constant>();
    auto stack = ov::pass::pattern::wrap_type<ov::op::util::FrameworkNode>(stack, axis);
```
As I can see you expect `ListConstruct` -> `stack` <- `Constant` pattern. It would make sense to match this exact pattern.